content/release-notes/1.20.0.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-10-06
+linktitle: "1.20.0"
+title: "1.20.0"
+kubernetes: "1.16, 1.17, and 1.18"
```

Comment:
Can't we add 1.19 now that kURL is out? And if we remove 1.16, do we have to put a deprecation notice somewhere?